python/src/aiconfig/editor/server/server.py

```diff
@@ -57,7 +58,7 @@
 CORS(app, resources={r"/api/*": {"origins": "*"}})
 
 
-def run_backend_server(edit_config: EditServerConfig) -> Result[str, str]:
+def run_backend_server(edit_config: EditServerConfig, cli_config_path: str) -> Result[str, str]:
```

Comment:
These two names are confusing since at surface level they seem to be very similar functionality. Can we call `cli_config_path` something like `settings.yaml`, or modify `edit_config` to be something more specific like `server_settings`?